server/services/team/service.ts

```diff
@@ -216,4 +216,12 @@ export class TeamService extends ServiceClass implements ITeamService {
 			_id: teamId,
 		}, { projection: { usernames: 0 } });
 	}
+
+	async deleteById(teamId: string): Promise<boolean> {
+		return !!await this.TeamModel.deleteOneById(teamId) || false;
```

Comment:
I think it's not needed to have the `|| false` in here, since `deleteOneById` will always return something (either if something was deleted or not). wdyt?